### DISCUSSION TITLE
Time picker for date control element

### DISCUSSION THREAD

**Discussion by s-wel** [Avaiga/taipy#162] [Ideas]
The date control element ([Docu](https://docs.taipy.io/en/latest/manuals/gui/viselements/date/#using-the-full-date-and-time)) visualizes the date and time information, but it seems there is no time picker. I suggest adding a time picker besides the date picker if the property "with_time" is True.

**Comment by FlorianJacta** [comment]
The time picker exists in the previous version of Taipy but has disappeared as we are in sync with the Javascript components we use. We constantly use their latest and most stable version. We will get back to you as soon as possible!

**Comment by FlorianJacta** [comment]
Related issue created on this subject: https://github.com/Avaiga/taipy-gui/issues/768

**Comment by s-wel** [comment]
Thanks. Great to hear it is on the todo list.
